src/ui/Loader.tsx

```diff
@@ -0,0 +1,78 @@
+import { motion } from "framer-motion";
+
+const spinAnimation = {
+  loop: Infinity,
+  ease: "linear",
+  duration: 1,
+};
+
+export function CircleLoader() {
+  return (
+    <div className='flex h-full flex-col items-center justify-center space-y-4'>
+      <div className='relative box-border h-8 w-8'>
+        <motion.span
+          className='absolute top-0 left-0 box-border block  h-8 w-8 rounded-full border-4 border-t-4 border-solid border-[#1A4670] border-t-[#1C7ED6]'
+          animate={{ rotate: 360 }}
+          transition={spinAnimation}
+        />
+      </div>
+    </div>
+  );
+}
+
+const containerVariants = {
```

Comment:
@ID-JA I think you should add these animation constants to the code preview, So people who will try to copy the code won't be stuck with where's the animations constants but will find everything in front of them, Please add only the ones that are used in the component

Reply:
thanks for your feedback, I fixed the problem.